.dockerignore

```diff
@@ -117,3 +117,6 @@ dynamodb-data/
 redis-data/
 node_modules/
 ui/node_modules/
+
+# Frontend Ignores. Local work would copy into the docker image and break things
```

Comment:
This shouldn't be necessary, but I am not sure if it causes any harm, as `yarn --cwd ui build:prod` will do `rm -rf ../consoleme/templates` prior to `yarn build`

Reply:
ok, I can undo this. I don't believe it will cause harm, but also doesn't hurt to ignore it since the dockerfile was rearranged